tools/telemetry-generator/package.json

```
@@ -35,13 +35,20 @@
 		"applicationinsights": "^2.4.1"
 	},
 	"devDependencies": {
+		"@fluid-internal/mocha-test-setup": "~2.0.5",
 		"@fluidframework/build-common": "^2.0.1",
 		"@fluidframework/eslint-config-fluid": "^5.2.0",
+		"@types/mocha": "^9.1.1",
```

Since this package is not part of the client release group, it doesn't have to match the dependency versions of everything there. It's better if we match the types packages to the versions of the dependencies.
```suggestion
		"@types/mocha": "^10.0.9",
```